spec/controllers/organizations_controller_spec.rb

```diff
@@ -1,5 +1,66 @@
 require 'rails_helper'
 
 RSpec.describe OrganizationsController, type: :controller do
+  let (:unpublished_organization) do
+    Organization.create(FactoryGirl.attributes_for(:organization).merge({published: false}))
+  end
 
+  let (:published_organization) do
+    Organization.create(FactoryGirl.attributes_for(:organization).merge({published: true}))
+  end
+
+  let (:admin_user) { User.create(FactoryGirl.attributes_for(:user).merge({admin: true})) }
```

Comment:
Follow the instructors here to make factories for `admin_user` and `published_organizations`
https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md#defining-factories 

You should also use `FactoryGirl.create` rather than `User.create` with `FactoryGirl.attributes_for`